locales/fil/json.json

```diff
@@ -964,7 +964,7 @@
     "Update": "Update",
     "Update & Continue Editing": "I-update at Ipagpatuloy ang Pag-edit",
     "Update :resource": "Update :resource",
-    "Update :resource: :title": "Update :resource: :title",
+    "Update :resource: :title": "Update :resource::title",
```

Comment:
Is it correct to delete this space?

Reply:
No, this is not correct. That's why I close this PR every day 🙈

I have not yet added to the translator the function of ignoring some values, where we will add what he always makes mistakes.